CODEOWNERS

```diff
@@ -1 +1 @@
-* @damacus @scopenco 
+* @damacus @scopenco @sous-chefs/grafana
```

Comment:
You can remove `@damacus` and `@scopenco`

Reply:
Done, but neither of you are in the group fyi